vector/src/main/java/im/vector/app/features/settings/devices/v2/VectorSettingsDevicesViewNavigator.kt

```diff
@@ -25,4 +26,8 @@ class VectorSettingsDevicesViewNavigator @Inject constructor() {
     fun navigateToSessionOverview(context: Context, deviceId: String) {
         context.startActivity(SessionOverviewActivity.newIntent(context, deviceId))
     }
+
+    fun navigateToOtherSessions(context: Context) {
```

Comment:
We should add a new unit tests for this method.

Reply:
Done.